### PR TITLE
fix: Update colors in AudioRecordViewController

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we update the colors for AudioRecordViewController and its associated views to match the [design specs](https://www.figma.com/file/exxrpYZ28N1WOylyw9TYh1/Accessibility-Quick-Wins-(iOS)?type=design&node-id=9-810&t=UfNT1HbAfT69kV2R-0)

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by KaterinaWire** [comment]
@agisilaos Why does this PR contain snapshots for new conversation (not only for audioRecord view)?

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/RecordingDotView.swift [hunk 1]
```diff
@@ -1,61 +1,69 @@
...
+    // MARK: - Methods
 
-    fileprivate func startAnimation() {
+    private func startAnimation() {
         self.alpha = 0
         delay(0.15) {
             UIView.animate(withDuration: 0.55, delay: 0, options: [.autoreverse, .repeat], animations: {
                 self.alpha = 1
-                }, completion: .none)
+            }, completion: .none)
```
Comment (netbe) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/RecordingDotView.swift]: can't you remove the completion as default param is nil?

**Review by netbe — APPROVED** [review]
LGTM;) just small suggestion

**Comment by agisilaos** [comment]
> @agisilaos Why does this PR contain snapshots for new conversation (not only for audioRecord view)?

@KaterinaWire This is not the case anymore.

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift [hunk 2]
```diff
@@ -41,11 +41,11 @@ final class AudioButtonOverlay: UIView {
...
     fileprivate lazy var widthConstraint: NSLayoutConstraint = widthAnchor.constraint(equalToConstant: initialViewWidth)
 
-    let darkColor = UIColor.from(scheme: .textForeground)
-    let brightColor = UIColor.from(scheme: .textBackground)
-    let greenColor = SemanticColors.LegacyColors.strongLimeGreen
-    let grayColor = UIColor.from(scheme: .audioButtonOverlay)
-    let superviewColor = UIColor.from(scheme: .background)
+    let darkColor = SemanticColors.Label.textDefault
+    let brightColor = SemanticColors.View.backgroundDefault
+    let greenColor = SemanticColors.Icon.backgroundJoinCall
```
Comment (johnxnguyen) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift]: In my opinion this is not the correct way to use semantic colors: the semantics of `backgroundJoinCall` is about the calling feature, and the audio button here is not related to the calling feature. So if you want to change the color of the call join button, you now will inadvertently change the color of the audio button. The problem here is they don't have the same semantics.

I see two ways forward:
- change the semantics of `backgroundJoinCall` to something like `positiveAction`
- create new semantics for the audio button, even if it has the same value as the join call button

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift]: Done!

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift [hunk 3]
```diff
@@ -41,11 +41,11 @@ final class AudioButtonOverlay: UIView {
...
 
-    let darkColor = UIColor.from(scheme: .textForeground)
-    let brightColor = UIColor.from(scheme: .textBackground)
-    let greenColor = SemanticColors.LegacyColors.strongLimeGreen
-    let grayColor = UIColor.from(scheme: .audioButtonOverlay)
-    let superviewColor = UIColor.from(scheme: .background)
+    let darkColor = SemanticColors.Label.textDefault
+    let brightColor = SemanticColors.View.backgroundDefault
+    let greenColor = SemanticColors.Button.backgroundAudioMessageOverlay
+    let grayColor = SemanticColors.View.backgroundUserCell
```
Comment (johnxnguyen) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift]: Also this seems like a misuse of a semantic color.

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordKeyboardViewController.swift [hunk 4]
```diff
@@ -191,27 +191,27 @@ final class AudioRecordKeyboardViewController: UIViewController, AudioRecordBase
...
         self.stopRecordButton.addTarget(self, action: #selector(stopRecordButtonPressed), for: .touchUpInside)
-        self.stopRecordButton.setBackgroundImageColor(SemanticColors.LegacyColors.vividRed, for: .normal)
+        self.stopRecordButton.setBackgroundImageColor(SemanticColors.Icon.foregroundDefaultRed, for: .normal)
         self.stopRecordButton.layer.masksToBounds = true
 
         self.confirmButton.setIcon(.checkmark, size: .tiny, for: [])
         self.confirmButton.setIconColor(.white, for: [])
         self.confirmButton.addTarget(self, action: #selector(confirmButtonPressed), for: .touchUpInside)
-        self.confirmButton.setBackgroundImageColor(SemanticColors.LegacyColors.strongLimeGreen, for: .normal)
+        self.confirmButton.setBackgroundImageColor(SemanticColors.Icon.backgroundJoinCall, for: .normal)
```
Comment (johnxnguyen) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordKeyboardViewController.swift]: Also this seems like a misuse of a semantic color.

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordKeyboardViewController.swift]: Done!

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift (hunk 3 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioButtonOverlay.swift]: Done!

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/RecordingDotView.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/RecordingDotView.swift]: Done!

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**